libtransact/src/scheduler/mod.rs

```diff
@@ -104,6 +104,15 @@ pub enum ExecutionTaskCompletionNotification {
     Valid(ContextId, String),
 }
 
+#[derive(Debug)]
+pub enum SchedulerError {}
+
+impl std::fmt::Display for SchedulerError {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        Ok(())
```

Comment:
This should still actually write something.

Reply:
Fixed